README.md

```diff
@@ -31,7 +31,8 @@ Please see the [MongoDB website](http://docs.mongodb.org/ecosystem/tutorial/ruby
 License
 -------
 
-Copyright (c) 2009-2017 Durran Jordan
+Copyright (c) 2009-2016 Durran Jordan
```

Comment:
There were no commits from Durran in 2017, the bump from 2016 to 2017 was made by a contributor.